sdk/src/assertions/actions.rs

```diff
@@ -109,6 +109,9 @@ pub struct Action {
     #[serde(skip_serializing_if = "Option::is_none")]
     changed: Option<String>,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
+    description: Option<String>,
```

Comment:
I think whatever test case we have that has this field in it should be removed. We should never have supported a this field here. It seems wrong to add this just to fix an incorrect test case.